### ISSUE TITLE
Add setting for middle click to open link in a new tab

### ISSUE THREAD

**Issue by kupiqu** [QupZilla/qupzilla#2216]
Not sure if this is implemented already but I could not find the way to do it.

If not implemented, please consider. I use it a lot in other browsers (Konqueror, Firefox and Chromium).

**Comment by cranes-bill** [comment]
Tthis is a **X** feature and works in QupZilla. Perhaps you are for [here](https://github.com/QupZilla/qupzilla/issues/2216).

**Comment by kupiqu** [comment]
I guess you meant #1909. 

Yes, that seems indeed to be my case, however I use v2.1 and, while double middle click is completely reliable, just a single click is very buggy: Its behavior its quite random, with a high failure rate.

I'll check my evdev rules to see if this can improve while keeping the functionality in my system.

**Comment by Digitalone1** [comment]
Hi everyone. I got the same issue.
Single click on middle button doesn't open a link in a new tab. But I discovered it works if I do double click. 
But single click works if I do it on a link located in the bookmark bar.
Is this behavior normal? I would like to open any link (located in the page or in the bookmark bar) in a new tab with a single click. 

I run Qupzilla on a laptop with Arch Linux x64. Input driver is libinput and middle click button is emulated with two finger click. It works on other browsers like Chromium and Firefox, but with Qupzilla I have to do double click with two fingers on the touchpad (unless the link is in the bookmark bar).

**Comment by tschlotfeldt** [comment]
Hi, I just ran in the same problem. It's the first time I'm trying QupZilla. Version is 2.1.1 with QtWebEngine-Version 5.8.0 under KDE and Arch Linux. Computer is a ThinkPad X230. The middle mouse click is not working but middle double click does.

Interestingly is the  middle click working when using the touchpad with three finger touch configured as middle click.

xev dosn't show any differenz:

**middle mouse button:**
```
ButtonPress event, serial 40, synthetic NO, window 0x9200001,
    root 0xd9, subw 0x0, time 55901286, (2,156), root:(834,185),
    state 0x0, button 2, same_screen YES

ButtonRelease event, serial 40, synthetic NO, window 0x9200001,
    root 0xd9, subw 0x0, time 55901286, (2,156), root:(834,185),
    state 0x200, button 2, same_screen YES

``` 

**Three finger click on touch pad configured as middle mouse click:**
```
ButtonPress event, serial 40, synthetic NO, window 0x9200001,
    root 0xd9, subw 0x0, time 55903594, (2,156), root:(834,185),
    state 0x0, button 2, same_screen YES

ButtonRelease event, serial 40, synthetic NO, window 0x9200001,
    root 0xd9, subw 0x0, time 55903694, (2,156), root:(834,185),
    state 0x200, button 2, same_screen YES

```

**Comment by joseriosneto** [comment]
I believe this issue was already addressed in #2238.

**Comment by kupiqu** [comment]
Indeed. This has just been fixed!
